playground.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
```

Comment:
Please remove this file (which includes the Benchmark class). Instead add some unit tests to the tests directory with appropriate asserts (see examples in test_minimizer.py)